app/components/ui/Translate.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Helmet } from 'react-helmet-async';
+
+// This is a prototype component for testing out the Google Translate widget. The logic below
+// dictates that this component only be displayed on staging; i.e. the widget will be displayed IF
+// location.hostname === 'staging.sfserviceguide.org'. To test locally point
+// staging.sfserviceguide.org to your local host IP in your /etc/hosts file; i.e.,
+// 127.0.0.1       staging.sfserviceguide.org
+// Then, navigate to http://staging.sfserviceguide.org:8080 to see the widget.
```

Comment:
Might be easier to test this by just modifying the code to be `if (true)`. Browsers might complain if you try to visit a site with http that you've visited in the past with https, since a common attack is to downgrade a secure connection to a non-secure one and to man-in-the-middle the user.